pixi.toml

```diff
@@ -0,0 +1,105 @@
+[project]
+name = "MSS"
+channels = ["conda-forge"]
+platforms = ["linux-64", "osx-64", "osx-arm64", "win-64"]
+
+[tasks]
+
+[dependencies]
+PyMySQL = ">=0.9.3"
+basemap = ">=1.3.3"
+cftime = ">=1.0.1"
+chameleon = "*"
+defusedxml = "*"
+email_validator = "*"
+execnet = "*"
+fastkml = ">=0.11"
+flask = ">=2.3.2"
+flask-cors = "*"
+flask-httpauth = "*"
+flask-login = "*"
+flask-mail = "*"
+flask-migrate = "*"
+flask-socketio = ">=5.1.0"
+flask-sqlalchemy = ">=3.0.0"
+flask-wtf = "*"
+fs_filepicker = "*"
+git = "*"
+gitpython = "*"
+gpxpy = ">=1.4.2"
+hdf4 = "*"
+isodate = "*"
+itsdangerous = "*"
+keyring = "*"
+libtiff = "<4.5.0"
+lxml = "*"
+markdown = "*"
+matplotlib = ">=3.5.3"
+menuinst = "*"
```

Comment:
menuinst is not implemented for pixi, https://github.com/prefix-dev/pixi/discussions/1894